pkg/grafana/client.go

```diff
@@ -51,7 +51,9 @@ func NewClient(baseURL *url.URL, cfg gapi.Config) (*Client, error) {
 func (c *Client) LookupUser(loginOrEmail string) (gapi.User, error) {
 	user, err := c.client.UserByEmail(loginOrEmail)
 	if err != nil {
-		if !strings.Contains(err.Error(), "User not found") {
+		// the error message is longer but given that this is constrained
+		// to a returned message from an specific call, it's better to keep it short
+		if !strings.Contains(err.Error(), "not found") {
```

Comment:
I like the specific error message, given grafana doesn't seem to have a well defined structure. 
An alternative would be to make all the strings lowercase.

Reply:
Indeed Grafana's error message are not that consistent I think. Do we want to support both lower and upercase cases?

Reply:
No, if we lower the whole error message then we are always comparing a deterministic case.